ubuntu/app-center
# (new) Added offline page

# Description
This pull request introduces a new feature: the "Offline Page." In situations where the user's device loses internet connectivity, this page will be displayed, providing a better user experience and useful information.

# Changes Made
- Created a new "offline_page.dart" file.
- Designed a user-friendly and informative offline page with appropriate styling.
- Implemented a mechanism to detect and trigger the display of the offline page when the user is offline.
- Included relevant assets and resources to enhance the offline page's appearance and functionality.

# How to Test
To verify the changes, follow these steps:

1. Open the app.
2. Disable your internet connection or put your device in Airplane Mode.
3. Observe that the "Offline Page" is displayed.
4. Ensure the page provides helpful information and has a pleasing design.
5. Re-enable your internet connection to see if the offline page disappears when the connection is restored.

# Screenshots
[![JFishTQ.md.png](https://iili.io/JFishTQ.md.png)](https://freeimage.host/i/JFishTQ)
